### DISCUSSION TITLE
Resizing simple montages

### DISCUSSION THREAD

**Discussion by RedactedCode** [ImageMagick/ImageMagick#4521] [Digital Image Processing]
I sometimes make a simple montage of images for tweets.

The final image almost always needs a resize.

I've been doing this (bash, IM 7.0.9):

`
montage image1.png image2.png -background (somecolour) -geometry +1+1 -tile (sometilespec) - | convert - -resize (someresizespec) image3.png
`

It works fine but I wondered if there was a "better" way?

**Comment by urban-warrior** [comment]
Your resize your canvas as you suggest.  You can also change the size of the rendered tiles with the `-geometry` option, e.g.,

```
montage -geometry 300x300 rose: wizard: montage.png
```

You can also try this command:

```
magick '*.png' -resize 320x320 -label %f ashlar:ashlar.png
```
